server/main.py

```diff
@@ -274,42 +252,26 @@ def api_ranking(dcid):
     result = collections.defaultdict(list)
     # TODO(boxu): make the stats_vars in a config.
     for parent in selected_parents:
-        # Population ranking
-        result['Population'].append({
-            'name': parent_names[parent],
-            'data': get_related_place(
-                dcid, 'Person', 'count', 'measuredValue',
-                same_place_type=True, within_place=parent)})
-        # Median income
-        result['Median Income'].append({
-            'name': parent_names[parent],
-            'data': get_related_place(
-                dcid, 'Person', 'income', 'medianValue',
-                pvs_string='age^Years15Onwards^incomeStatus^WithIncome',
-                same_place_type=True, within_place=parent)})
-        # Median age
-        result['Median Age'].append({
-            'name': parent_names[parent],
-            'data': get_related_place(
-                dcid, 'Person', 'age', 'medianValue',
-                same_place_type=True, within_place=parent)})
-        # Unemployment rate
-        result['Unemployment Rate'].append({
-            'name': parent_names[parent],
-            'data': get_related_place(
-                dcid, 'Person', 'unemploymentRate', 'measuredValue',
-                same_place_type=True, within_place=parent)})
+        stats_var_string = '^'.join(RANKING_STATS.keys())
+        response = get_related_place(
+            dcid, stats_var_string, same_place_type=True, within_place=parent)
+        for stats_var, data in response.items():
+            result[RANKING_STATS[stats_var]].append(
+                {'name': parent_names[parent], 'data': data})
+
         # Crime
-        result['Crime per capita'].append({
-            'name': parent_names[parent],
-            'data': get_related_place(
-                dcid, 'CriminalActivities', 'count', 'measuredValue',
-                pvs_string='crimeType^UCR_CombinedCrime',
-                same_place_type=True, within_place=parent, is_per_capita=True)})
-
-    result['label'] = [
-        'Population', 'Median Income', 'Median Age', 'Unemployment Rate',
-        'Crime per capita']
+        crime_statsvar = {
+            'Count_CriminalActivities_CombinedCrime': 'Crime per capita'}
```

Comment:
Should this be part of RANKING_STATS as well?

Reply:
This one uses perCapita option while the rest does not. The API does not support mixed options, so need to make a separate call. Made an issue to improve on mixer side: https://github.com/datacommonsorg/mixer/issues/262

Reply:
I see, thanks for the clarification. Could you add a note to that here (or in RANKING_STATS above)

Reply:
Done